044_bb_L1118/044_bb_L1118_unsafe.py

```
@@ -0,0 +1,2833 @@
+    return necessaries_retries
+
+
+def start_process(command):
+    """Start rsync commands
+
+    :param command: rsync list command
+    :return: command
+    """
+    fd = get_std_out()
+    if fd == "STDOUT":
+        p = subprocess.call(command, shell=True)
```

Ruff (0.14.2)

`subprocess` call with `shell=True` identified, security issue

(S602)